packages/patternfly-4/react-inline-edit-extension/package.json

```diff
@@ -28,8 +28,8 @@
   "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/",
   "dependencies": {
     "@patternfly/patternfly": "2.65.0",
-    "@patternfly/react-core": "^3.140.6",
```

Comment:
The latest prerelease for react-core is 3.140.6, so I believe this needs to stay as-is?